ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml

```diff
@@ -41,13 +47,24 @@ tests:
   cron: '@yearly'
   steps:
     cluster_profile: gcp-qe
+    dependencies:
```

Comment:
You are using this job for pj-rehearse testing on purpose? For online troubleshooting?

Reply:
Yes, it's true. I could recover it too.